ReactNativeClient/lib/models/Folder.js

```diff
@@ -218,6 +218,17 @@ class Folder extends BaseItem {
 		return output;
 	}
 
+	static async expandTreeOnGoto(folders, folder) {
```

Comment:
Not sure why "OnGoto". Please rename to "expandTree". Also you only need the folder parent ID in this function and it's good practice to pass to a function only what you need. So it could just be `static async expandTree(folders, folderId)`. And you call it like so `Folder.expandTree(this.props.folders, this.state.folder.parent_id)`

Reply:
I'll keep this in mind. I've made the requested changes and wrote a test for folderPath. Please review.